app/repository/sbrctrl/RestAdminDataUnitLinksEditRepository.scala

```diff
@@ -6,16 +6,46 @@ import com.typesafe.scalalogging.LazyLogging
 import repository.DataSourceNames._
 import repository._
 import repository.rest.Repository
-import uk.gov.ons.sbr.models.{ Period, VatRef }
+import uk.gov.ons.sbr.models.UnitType.{ LegalUnit, ValueAddedTax }
+import uk.gov.ons.sbr.models.{ Period, UnitId, UnitKey, VatRef }
 import uk.gov.ons.sbr.models.edit.Patch
-import unitref.VatUnitRef
 
 import scala.concurrent._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+case class UrlAndPatch(url: String, patch: Patch)
+object UrlAndPatch {
+  def apply(urlAndPatch: (String, Patch)): UrlAndPatch = UrlAndPatch(urlAndPatch._1, urlAndPatch._2)
```

Comment:
If you treat the existing apply function as a function, it will have a tupled method - which lets you invoke the function with a tuple rather than 2 individual arguments.

Reply:
That's handy!